kedro-datasets/static/jsonschema/kedro-catalog-0.19.json

```diff
@@ -0,0 +1,1470 @@
+{
+    "type": "object",
+    "patternProperties": {
+      "^[a-z0-9-_]+$": {
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "CachedDataset",
```

Comment:
CachedDataset, MemoryDataset and LambdaDataset are still part of the Kedro core package, so I'm not sure they should actually be part of this list..